Source/Core/DolphinQt2/HotkeyScheduler.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <thread>
+
+class MainWindow;
+
+class HotkeyScheduler
+{
+public:
+  explicit HotkeyScheduler(MainWindow* parent);
+
+  void Start();
+  void Stop();
+
+private:
+  void Run();
+
+  bool m_stop_requested;
```

Comment:
You're reading/writing to this bool from two different threads though, so this should be a std::atomic<bool> or even better, a Common::Flag (Common/Flag.h).